src/Styleguide/Pages/Artwork/Sidebar.tsx

```diff
@@ -0,0 +1,56 @@
+import React, { Component } from "react"
+import styled from "styled-components"
+import { RulerSeparator } from "../../Elements/RulerSeparator"
+import { Artists } from "./Sidebar/Artists"
+import { ArtworkMetadata } from "./Sidebar/ArtworkMetadata"
+import { Commercial } from "./Sidebar/Commercial"
+
+export interface ArtworkSidebarProps {
```

Comment:
Created this sidebar component to hold all the other building blocks for the sidebar. This props I think will be what we'll get from MP. And this component will be responsible for rendering proper parts depending is the artwork is regular artwork or auction artwork.

Reply:
It looks like you may be doing a great job on defining the types inline with what relay-compiler will eventually generate, but just be aware that once we make these components dynamic and fetch data the generated types might render a bunch of this work unnecessary. So if you’re spending a lot of time on it, then you might wanna do it less rigorously.